docs/glues/herbs2gql.md

```diff
@@ -130,6 +130,17 @@ const resolverFunc = () => { }
 const [gql, resolver] = usecase2subscription(usecase, resolverFunc)
 ```
 
+### Herbarium to graphlq
```

Comment:
>Herbarium to graphlq

1. Suggestion: Rename to Herbs to GraphQL

2. Since this kind of integration will be the primary way of converting Herbs to GraphQL, this should be the first method to show, not the last

Reply:
It could make the documentation not clear, I'm already inside the Herbs to graphql doc, why have a title "herbs to graphlq" inside the "herbs to graphql" doc.
Use the herbrarium's name in the title is useful, maybe we could rename it to "using herbrarium" or something like this

Reply:
> why have a title "herbs to graphlq" inside the "herbs to graphql" doc.

makes sense. however, I would like to avoid the highlight on Herbarium... 
Maybe this section should be just on the `### Using` section and the others (older ones) could be on the `###Advanced` section